app/page.tsx

```diff
@@ -9,14 +9,39 @@ export default function Page() {
       <PageTitle>Department of Decentralization</PageTitle>
       <div className="prose max-w-none pb-8 pt-8 dark:prose-invert">
         <div>
-          The Department of Decentralization is a collective of people from various crypto,
-          decentralization and peer-to-peer communities in and around Berlin. The group assembled in
-          2018 to organize ETHBerlin and has been active since.
+          The Department of Decentralization is a collective of people from the cryptocurrency,
+          decentralization, and peer-to-peer community in Berlin. The group assembled in 2018 to
```

Comment:
```suggestion
          The Department of Decentralization is a collective of people from the
          decentralization community in Berlin. The group assembled in 2018 to
```

I would not let 2 sub-eleents of decentralisation stick out like that and especially not let cryptocurrency be first in the list

Reply:
Then it reads literally "the decentralization collective is people of decentralization."

What about _"The Department of Decentralization is a collective of people from the cryptocurrency and peer-to-peer community in Berlin."_ instead?

Reply:
How about:

> The Department of Decentralization is a collective of people in Berlin.

simplify and do not box us too much ^^